tests/testthat/_snaps/ext_ser_call.md

```diff
@@ -0,0 +1,14 @@
+# ext_ser_call()
+
+    Code
+      ext_ser_call(pp, "x")
+    Output
+      [1] "x <- import.ts(file.dat)"
```

Comment:
@christophsax, I am unsure about this generated call. Does it sound reasonable to you? Why not something like `x <- import.ts(file.dat, format = \"datevalue\", start = c(2014, 1), frequency = 12, name = \"xyz\")`?